src/frontends/pytorch/src/op/pad.cpp

```diff
@@ -134,6 +134,14 @@ OutputVector translate_reflection_pad_nd_fx(const NodeContext& context) {
     return translate_pad_common(context, data, paddings, pad_value, "reflect");
 }
 
+OutputVector translate_replicate_pad_nd_fx{const NodeContext & context} {
+    num_inputs_check(context, 2, 2);
+    auto data = context.get_input(0);
+    auto paddings = context.const_input<std::vector<int64_t>>(1);
+    Output<Node> pad_value = context.mark_node(v0::Constant::create(element::f32, Shape{}, {0}));
+    return translate_pad_common(context, data, paddings, pad_value, "replicate");
```

Comment:
I am not sure that logic from `translate_pad_common` is suitable for `replication` operation.
That is because a format of paddings is as follows `<pad_begin_axis1>, <pad_end_axis1>, <pad_begin_axis2>, <pad_end_axis2>, ...` in case of tuple.

May be we need to do as follows:
1. do broadcast of padding to convert to `tuple` case
2. create a pad vector with zeros with length equal to rank of `input`
3. scatter elements from broadcasted padding to `pad` vector with known indices defined for each 1d, 2d and 3d cases

Reply:
I am not able to currently get why  this logic maybe incorrect can you please give an example to explain. it would be very helpful. Thank you

Reply:
Hi @11happy, 
I meant logic for preparation of pad vector can be simplified but I understand it may be out-of-scope for this PR because it is used by other translators.
Let us see tests if they pass.  please resolve conflicts and I will trigger CI for your PR.

Thanks.

Reply:
resolved the conflicts.
Thank you